diff.py

```diff
@@ -3073,9 +3079,11 @@ def align_diffs(
         diff_lines = [
             (base, new, old if old != new else empty) for base, new, old in diff_lines
         ]
-    elif config.zeroway or config.oneway:
+    elif config.diff_type in [DiffType.SINGLE or DiffType.SINGLE_BASE]:
```

Comment:
```suggestion
    elif config.diff_type in (DiffType.SINGLE, DiffType.SINGLE_BASE):
```
(() or {} being slightly more idiomatic than [] I think? but I don't remember where I got that from)

Reply:
not sure why I put "or" there..